### ISSUE TITLE
json problem

### ISSUE THREAD

**Issue by virgamadalina** [HashLips/hashlips_art_engine#953]
i generated 70 png and i have also json for them but i have some extra json
exeple: 1.png.     1.json
             2 png.    2.json
                            2 2.json
             3 png.      3 json
             4 png     4 2 json 

what i have to do?

**Comment by bolshoytoster** [comment]
@virgamadalina I don't know how they were made (what commands have you run & what version of hashlips are you using) but you can probably just delete them.

**Comment by virgamadalina** [comment]
yes. i can do that but i want to have 5555 and it is hard to check it all

 1.1.2 version

**Comment by bolshoytoster** [comment]
@virgamadalina you can delete all of the files apart from `_metadata.json` then run `node utils/update_metadata.js` to regenerate them.

Also, what commands have you run?

**Comment by virgamadalina** [comment]
im very noob 
i follow the steps. 1.download zip code
2. nmp install
3. i configued all the layers,i saved it and after 
4. node index.js
5. nest step should be to node utils update info after i have a cid
6.

**Comment by virgamadalina** [comment]
i run it again and this time everthing was ok. now im trying to list the test collection on devnet tosee how it looks
  thanks for your time
